boards/arm/nucleo_l4r5zi/nucleo_l4r5zi.yaml

```diff
@@ -17,5 +17,6 @@ supported:
   - usb device
   - nvs
   - counter
+  - lptim
```

Comment:
Was this change intended to be included in this PR?

Reply:
mismatch, 
Removed